### DISCUSSION TITLE
How can I know the phase of the Moon?

### DISCUSSION THREAD

**Discussion by Laurent-Laveder** [Stellarium/stellarium#3636] [Q&A]
Hi!
with a previous version of Stellarium, the phase of the Moon was available among the informations. But now, with the lastest version, I can't find it! Is there someone to tell me where to find it?
Thanks a lot.

**Comment by gzotti** [comment]
Make sure you have "additional information" ticked in the information options.

**Comment by Laurent-Laveder** [comment]
Thank you @gzotti but I forget to write that I would like the phase in % (not in days) as I had in the previous version. Is it possible?

**Reply by gzotti** [reply]
Then activate Elongation info. It is included in that group. Play around with the other settings or activate "All" to see what's available.

**Comment by Laurent-Laveder** [comment]
Wonderfull! Elongation gives me the info needed. Thank you so much!
